addons/mail/static/src/new/discuss/discuss.xml

```diff
@@ -3,7 +3,7 @@
 
 <t t-name="mail.discuss" owl="1">
     <div class="d-flex h-100 flex-grow-1">
-        <Sidebar />
+        <t t-if="!props.hideSidebar"><Sidebar/></t>
```

Comment:
- rename `hideSidebar` to just `sidebar` (`Boolean`, default: `true`)
- Can do `<Sidebar t-if="props.sidebar"/>` (shorter)

Also document props of `Discuss` rather than keep `*`